front/lib/resources/content_fragment_resource.ts

```diff
@@ -365,13 +367,69 @@ async function renderFromFileId(
   } else {
     const content = await getProcessedFileContent(workspace, fileId);
 
+    const messageWithFullContent = {
```

Comment:
Seen IRL, In practice, today's behavior is that if `fileId` exists, it indicates a recent file upload within the app. If it does not exist, it refers to older file uploads or those uploaded through the API.

I recommend standardizing the new CSV behavior for both our app and the public API. Can we encapsulate this logic in a dedicated function that can be used wherever needed?

Additionally, there's no need to add "(truncated)" here since it's already being appended during generation:
https://github.com/dust-tt/dust/blob/fb454e4988e5a55ab570563403efeb77d64e5d23/front/lib/api/assistant/generation.ts#L235